platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -152,6 +152,17 @@ NSExpression(forFunction: "lowercase:",
              arguments: [NSExpression(forKeyPath: "ISO 3166-1:2006")])
 ```
 
+If a feature attribute name contains a dot, the string can be wrapped in a
+`MGL_FUNCTION`. For a feature attribute named `us.state`, you could use:
```

Comment:
Consider linking to the `#code-mgl_function-code` section, since jazzy doesn’t autolink to sections of Markdown documents.